src/Form/UpdateMediaItemForVideoType.php

```diff
@@ -131,8 +132,8 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
       $this->submitFormProcessRequest($request, $guid, $success_text, $error_text);
       return;
     }
-    $guid_not_found_exception = new \Exception("Given GUID doesn't exist, please check and try again.");
-    $error_text = (string) $this->t("Given GUID doesn't exist, please check and try again.");
+    $guid_not_found_exception = new \Exception("Given guid doesn't exist, please check and try again.");
```

Comment:
Can we `sprintf` in the GUID into this string (and the one below?).

Instead of `\Exception`, perhaps RuntimeException or DomainException?